TEST.txt

```diff
@@ -0,0 +1,53 @@
+Four different scenarios can be tested with our DoS mitigation tool dostroy:
+1. Rate limiting
+2. Slowloris prevention
+3. RUDY prevention
+4. Generic error handling
+
+Requirements:
+- Node.js installed (≥ v8.15.0).
+- npm installed (≥ v6.10.2).
+- Preferrably two machine to run the tests on as behaviour can't be guaranteed when the attacker and target is the same computer.
```

Comment:
Preferrably -> Preferably

Reply:
machine -> machines

Reply:
I fixed all of the spelling errors in the second commit :)